### PR TITLE
Fix: use of Link tag instead of anchor tag.

### PR DESCRIPTION
**Pull request by its-kunal**
This pull request will fix the [issue](https://github.com/keploy/keploy/issues/1800)

- Change all anchor tags to Link tags.

### CODE REVIEW

**Comment by its-kunal** [comment]
Hey @nehagup please review the pull request.

**Comment by Hermione2408** [comment]
Hey @its-kunal please create a single PR for both link and Image tag

**Comment by its-kunal** [comment]
Hey @Hermione2408, merged changes from `Image` branch.

File: next.config.js [hunk 1]
```diff
@@ -14,4 +14,14 @@ module.exports = {
             //   },
         ];
     },
+    images: {
```
Comment (Hermione2408) [inline next.config.js:17]: @its-kunal why is this required?

**Comment by nehagup** [comment]
Hi @its-kunal are you still working on this?

**Comment by its-kunal** [comment]
@nehagup, I have successfully made the changes for the [issue](https://github.com/keploy/keploy/issues/1800). Please feel free to review it.

**Comment by Hermione2408** [comment]
> @nehagup, I have successfully made the changes for the [issue](https://github.com/keploy/keploy/issues/1800). Please feel free to review it.

Hey @its-kunal there are many warning in console related to images not being served over HTTPS and image aspect ratios being altered without maintaining the original proportions can you please check 
Screenshot from 2024-04-22 23-49-34

**Comment by its-kunal** [comment]
Thanks for the informative feedback, I will fix this soon.

**Comment by its-kunal** [comment]
After evaluation for these warnings, it occurs also in the `main` branch. A separate issue should be considered for this.
image

**Comment by its-kunal** [comment]
Hey @Hermione2408, made the changes to fix the warnings.

**Comment by Hermione2408** [comment]
> Hey @Hermione2408, made the changes to fix the warnings.

Hey @its-kunal still some warning is remaining can you please check?

**Comment by its-kunal** [comment]
@Hermione2408 fix for the warning.

**Comment by its-kunal** [comment]
These warnings also occur in the main branch, and a separate issue should be considered.
image

**Comment by Hermione2408** [comment]
LGTM @nehagup , tested locally build is working fine!

**Review by nehagup — APPROVED** [review]
thanks @its-kunal ! 🚀
